meta/ibm_zos_core_meta.yml

```diff
@@ -7,4 +7,4 @@ managed_requirements:
     -
         name: "Z Open Automation Utilities"
         version:
-            - "1.3.0"
+            - ">=1.3.0"
```

Comment:
Yes, this meta data is always updated during the release, its not used by this collection, its used by the playbook repository, there were plans for this but nothing ever went into effect but we keep it around anyhow since its easy to maintain.